chronologic/eth-alarm-clock-dapp
# [WIP] DApp Docker Image

Introduces a **Dockerfile** which can be used to set up a DApp Docker container.

Also:
- Removes the old dockerization workflow as it is no longer in use and holds no value

**TESTING PROCEDURE**
1. Build the image - `npm run build:docker`
2. Run the container - `npm run docker`
3. Visit `localhost:8080` in your browser and the DApp should be available as a Docker container
4. (Optional) To stop and remove the container, run `npm run docker:remove`
